tests/Fixtures/TestProject/config/config.yml

```diff
@@ -83,4 +83,9 @@ security:
             context:            user
             pattern:            /admin/(.*)
             http_basic:
-            anonymous:          false
+
+services:
+    _defaults:
+        public: true
+    annotation_reader:
```

Comment:
jaky to tu ma vyznam? nenasel jsem, ze by se nekde v kodu pouzival annotation reader.

Reply:
moje chyba, zapomnel jsem si aktualizovat zavislosti. ted uz je to jasne. nejlepsi reseni by bylo nahradit v testech anotace za atributy u `tests/Fixtures/TestProject/ImaticDataBundle/Entity`, ale to ted resit nemusime.